### ISSUE TITLE
Current PR doesn't appear on status bar when checked out from command line

### ISSUE THREAD

**Issue by jcansdale** [github/VisualStudio#1591]
- GitHub Extension for Visual Studio version: 
- Visual Studio version: All

#### Repro

1. Clone a new repository
2. Check out PR branch from command line
3. PR doesn't appear on the status bar

Also:

1. Clone a new repository
2. Create a PR using `Create New` on `GitHub` pane
3. PR doesn't appear on the status bar

This means that people who haven't used the extension aren't being guided towards extension features that would be relevant in the current context.

#### Simple mitigation

At the moment the PR icon and the PR number is a single button. We only show this button when we know it's a PR branch (the branch is labeled when opened in the PR detail view). Hence, the button only appears when the has already used the GitHub pane functionality.

![image](https://user-images.githubusercontent.com/11719160/38550498-9b0470fc-3cae-11e8-8f1a-abdf76e45211.png)

We could easily split the button into 2, where the PR icon opens the PR list and the PR number opens the individual PR. This would have 2 advantages.

1. It would keep the command for the PR icon consistent with the `GitHub` pane (opens the PR list)

![image](https://user-images.githubusercontent.com/11719160/38551286-b2919252-3cb0-11e8-8d35-54c168cda0a0.png)

2. It would guide the user towards the PR list when they're using a Git repo (when they can open or create a PR)

#### Proper fix

1. Label the current PR when the PR list is loaded (not when the PR is specifically opened). This would help guide the user towards the current PR after they open the PR list.
2. Label the PR when created using `Create New` on `GitHub` pane

**Comment by grokys** [comment]
Yes, this is a known issue. I'm hoping that we can solve this as we move to GraphQL.

**Comment by jcansdale** [comment]
@grokys what about when creating PRs with the extension? Should they be showing up?

**Comment by grokys** [comment]
Yes, actually we can fix that part already.
